src/pynwb/core.py

```diff
@@ -60,6 +60,26 @@ def prepend_string(string, prepend='    '):
     return prepend + prepend.join(string.splitlines(True))
 
 
+def list2str(v):
+    return str(np.array(v)) + '\n'
+
+
+""" This works for 1D arrays but not 2D arrays
```

Comment:
Which function object (e.g,. function) does this comment belong to, it seems to be a bit out of place in the middle of LabelledDict class?

https://github.com/NeurodataWithoutBorders/pynwb/blob/c1a616fd12c9e0e56fa847cbe607bc7511098ef1/src/pynwb/core.py#L67-L80

Reply:
this PR is a bit of a work-in-progress. That is code I was using to attempt to do the printing myself for datasets without converting them to numpy. It works well for 1D lists, but once you get into multidimensional arrays the problem gets a bit harder. I'd prefer to just convert to numpy on the fly whenever people request a print, since that library handles this problem very well, but I realize that's extra overhead we may not want. This code is my attempt to try to do it myself, and I didn't want to throw it away just yet in case the team wanted to do this the long way.